### PR TITLE
#542

### PR DESCRIPTION
**Pull request by Rita-Okonkwo**
I added the tabs vs space section in the vs code outline. Also resolved a merge conflict in the command line file

### CODE REVIEW

File: vs-code/vscode.md [hunk 1]
```diff
@@ -100,3 +100,23 @@ Apprentices make a cheat sheet (could be on paper or in a note on their Desktop)
 - Open a file side by side
 - Open integrated terminal
 
+### TABS vs SPACES
+The age-old tabs vs spaces argument is a very heated and hillarious one. Some developers argue that tabs are better for code indentation while others argue that spaces are better. Both arguments are valid with the following points:
```
Comment (manaswinidas) [inline vs-code/vscode.md]: hilarious in place of hillarious

Comment (Rita-Okonkwo) [reply vs-code/vscode.md]: Alright thanks. Done

File: command-line/dot-profile.md [hunk 2]
```diff
@@ -108,11 +108,7 @@ of helpful libraries are written using scripts in this way.
 
 #### Aliases
 
-It's often helpful to make commands for yourself that are short cuts. Bash aliases allows you to set a shortcut command for a longer command. For example what if you want to change to your directory but you don't want to type 
- ```bash
- cd ~/scripts
- ```
-What if you could just type `cdscr` instead? Aliases are defined in the .bash_profile or .bashrc file typically in your home directory. 
+It's often helpful to make commands for yourself that are short cuts. Bash aliases allows you to set a shortcut command for a longer command. For example what if you want to change to your directory but you don't want to type `cd ~/scripts`. What if you could just type `cdscr` instead? Aliases are defined in the .bash_profile or .bashrc file typically in your home directory.
```
Comment (alodahl) [inline command-line/dot-profile.md]: Why did this get changed?

**Comment by alodahl** [comment]
Please title your prs with a very short description of the changes. The description should contain the issue number and any more details you think we need to know.

**Comment by alodahl** [comment]
Heres a good example. https://github.com/Techtonica/curriculum/pull/637

File: command-line/dot-profile.md (hunk 2 above)
Comment (Rita-Okonkwo) [reply command-line/dot-profile.md]: Hi Alina, it had a merge conflict with the master branch. But I've resolved it 
Nothing got charged

**Pull request merged**